azure-pipelines/pull-request-validation/build-consumers.yml

```diff
@@ -210,10 +223,9 @@ stages:
           cwd: $(Build.SourcesDirectory)/broker-java-root
           tasks: LinuxBroker:clean LinuxBroker:assemble --build-cache --info
           publishJUnitResults: false
-          jdkArchitecture: x64
-          jdkVersionOption: "1.11"
+          jdkArchitecture: x86
+          sqAnalysisBreakBuildIfQualityGateFailed: false
```

Comment:
Why is this set to false? Don't we want to fail the PR merge if Sonar Qube analysis breaks?

Reply:
Copied from the standalone linux validation pipeline. I've removed it, might not be needed